projects/saturn/src/main/java/io/fairspace/saturn/services/metadata/MetadataApp.java

```diff
@@ -60,7 +61,7 @@ protected void initApp() {
         });
         patch("/", (req, res) -> {
             var model = deserialize(req.body(), req.contentType());
-            var doViewsUpdate = req.queryParamOrDefault(DO_VIEWS_UPDATE, FALSE.toString());
+            var doViewsUpdate = req.queryParamOrDefault(DO_VIEWS_UPDATE, TRUE.toString());
```

Comment:
there are few more places in the api where it is set to false (see delete and put)